.travis.yml

```diff
@@ -20,6 +20,7 @@ env:
   matrix:
     - ROS_DISTRO=hydro  USE_DEB=true
     - ROS_DISTRO=hydro  USE_DEB=false   NOT_TEST_INSTALL=true
+    - ROS_DISTRO=hydro  USE_DEB=false   NOT_TEST_INSTALL=true EXTRA_DEB="ros-hydro-roseus ros-hydro-euscollada ros-hydro-pr2eus" CATKIN_PARALLEL_JOBS='-p1'
```

Comment:
Frequently `ROS_DISTRO=hydro  USE_DEB=false   NOT_TEST_INSTALL=true EXTRA_DEB="ros-hydro-roseus ros-hydro-euscollada ros-hydro-pr2eus" ` fails because of compile error because of `kill` and memory shortage.

When I add `CATKIN_PARALLEL_JOBS='-p1'`, travis seems to be OK.